<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherInformationSystem</class>
 <widget class="QMainWindow" name="WeatherInformationSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>692</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>WeatherInformationSystem</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: #20AAF9;}
QPushButton{
background-color: #CCCCCC;
font-size: 19px;
border: 2px solid black;	
}
QListWidget{
background-color: #CCCCCC;
font-size: 19px;	
border: 2px solid black;
}
QLabel{
background-color: #CCCCCC;
font-size: 19px;	
border: 2px solid black;
}
QCheckBox{
background-color: #CCCCCC;
font-size: 19px;	
border: 2px solid black;
}
QSlider{
background-color: #CCCCCC;
font-size: 19px;	
border: 2px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="weather_res">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);
font: 22pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Input city name here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>GetWeather</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Clear Result</string>
    </property>
   </widget>
   <widget class="QListWidget" name="city_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="days_list">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeCity_Button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
